blade-graphics/src/lib.rs

```diff
@@ -307,6 +308,15 @@ pub enum TextureFormat {
     Bc4Snorm,
     Bc5Unorm,
     Bc5Snorm,
+    Bc6hUfloat,
+    Bc6hFloat,
+    Bc7Unorm,
+    Bc7UnormSrgb,
+    // packed 32-bit
+    Rgb10a2Unorm,
+    Bgr10a2Unorm,
```

Comment:
let's not add this one. It's not very useful, and it's not translating nicely

Reply:
removing